packages/fxa-auth-server/lib/metricsCache.ts

```diff
@@ -4,29 +4,43 @@
 
 'use strict';
 
-const { RedisShared } = require('fxa-shared/db/redis');
-const { Container } = require('typedi');
-const { AuthLogger } = require('./types');
-const { StatsD } = require('hot-shots');
+import { RedisShared, Config as RedisSharedConfig } from 'fxa-shared/db/redis';
+import { Container } from 'typedi';
+import { AuthLogger } from './types';
+import { StatsD } from 'hot-shots';
 
 function resolveLogger() {
   if (Container.has(AuthLogger)) {
     return Container.get(AuthLogger);
   }
+  return;
 }
 function resolveMetrics() {
   if (Container.has(StatsD)) {
     return Container.get(StatsD);
   }
+  return;
 }
 
+export type MetricsRedisConfig = {
+  redis: {
+    metrics: RedisSharedConfig & {
+      lifetime: number;
+    };
+  };
+};
+
 export class MetricsRedis extends RedisShared {
-  constructor(config) {
+  private enabled: boolean;
+  private lifetime: number;
+  private prefix: string;
+
+  constructor(config: MetricsRedisConfig) {
     super(config.redis.metrics, resolveLogger(), resolveMetrics());
-    this.enabled = config.redis?.metrics?.enabled || false;
-    this.config = config;
-    this.prefix = config.redis?.metrics?.prefix || 'metrics:';
-    this.lifetime = config.redis?.metrics?.lifetime || 7200;
+    this.enabled = config.redis.metrics.enabled || false;
+    // TBD - Assigned but not used?
+    this.prefix = config.redis.metrics.prefix || 'metrics:';
```

Comment:
Typecript picked up that `this.prefix` wasn't actually being used. Is that correct?

Reply:
Good catch, this can be removed. The prefix is passed in the `super` constructor above in the `config.redis.metrics` object.